gis/gdal/Portfile

```diff
@@ -37,10 +36,6 @@ homepage            https://www.gdal.org/
 master_sites        https://download.osgeo.org/gdal/${version}/
 use_xz              yes
 
-# this can be removed in a future version of MacPorts
-# see https://github.com/macports/macports-base/pull/161
-compiler.blacklist-append {clang < 800}
```

Comment:
I'm now inclined to revise this to `{clang < 802}`. There has since been a [compilation error observed on 10.11 (Xcode 8.2.1)](https://build.macports.org/builders/ports-10.11_x86_64-builder/builds/121236/steps/install-port/logs/stdio) which is not observed on 10.10 (which uses MacPorts' clang 9.0 instead):
```
ogrgeopackagetablelayer.cpp:300:50: error: call to 'abs' is ambiguous
                            const int TZOffset = std::abs(sField.Date.TZFlag - 100) * 15;
                                                 ^~~~~~~~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/cmath:664:1: note: candidate function
abs(float __lcpp_x) _NOEXCEPT {return fabsf(__lcpp_x);}
^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/cmath:668:1: note: candidate function
abs(double __lcpp_x) _NOEXCEPT {return fabs(__lcpp_x);}
^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/cmath:672:1: note: candidate function
abs(long double __lcpp_x) _NOEXCEPT {return fabsl(__lcpp_x);}
^

```

Reply:
Changes to `gdal` dropped from this PR; proposed revisions in a separate PR (#8545) to give more time to run on Travis CI.